filament/backend/include/private/backend/HandleAllocator.h

```diff
@@ -173,8 +174,10 @@ class HandleAllocator {
                 uint8_t const age = (tag & HANDLE_AGE_MASK) >> HANDLE_AGE_SHIFT;
                 auto const pNode = static_cast<typename Allocator::Node*>(p);
                 uint8_t const expectedAge = pNode[-1].age;
-                FILAMENT_CHECK_POSTCONDITION(expectedAge == age) <<
-                        "use-after-free of Handle with id=" << handle.getId();
+                // get_handle_tag() is only called if the check fails.
+                FILAMENT_CHECK_POSTCONDITION(expectedAge == age)
+                        << "use-after-free of Handle with id=" << handle.getId()
+                        << ", tag=" << get_handle_tag(handle.getId()).c_str_safe();
```

Comment:
I should have caught this before (sorry), but this should be `getHandleTag()`.

Reply:
Some of the other methods are snake case, so wasn't sure :)